tests/test_context.py

```diff
@@ -916,25 +914,24 @@ def get_template_data(self, args, kwargs, slots, context):
         assert child_data["dummy"] == "a1bc3f"
         assert parent_data["csrf_token"] == child_data["csrf_token"]
 
-    def test_raises_on_accessing_context_processors_data_outside_of_rendering(self):
+    def test_context_processors_data_outside_of_rendering(self):
```

Comment:
This test previously checked that we can't use Component instance outside of the component methods. Now it tests that we can 🎉